webpack.config.js

```diff
@@ -110,23 +101,29 @@ const config = [
     },
     output: {
       path: PATHS.DIST,
-      filename: 'styles/[name].css',
+      filename: 'styles/[name].[chunkhash].css',
+      chunkFilename: 'styles/[name].chunk.[chunkhash].css',
```

Comment:
I'm not 100% sure what's going on here, but I think the latest version of webpack wants to split CSS file in chunks. This bits controls where the chunks go.